x/interchainstaking/keeper/keeper.go

```diff
@@ -516,6 +511,9 @@ func (k *Keeper) EmitPerformanceBalanceQuery(ctx sdk.Context, zone *types.Zone)
 		100,
 	)
 
+	zone.PerformanceAddress.BalanceWaitgroup = 1
```

Comment:
intending to use, or eliminate concurrency?

Reply:
Not sure what you mean?

Reply:
balancewaitgroup?

this ins't a threading thing?